sympy/physics/quantum/qexpr.py

```diff
@@ -97,9 +97,9 @@ class QExpr(Expr):
     # The separator used in printing the label.
     _label_separator = ''
 
-    @property
-    def free_symbols(self):
-        return {self}
+    # @property
+    # def free_symbols(self):
+    #     return {self}
```

Comment:
It is good to remove these things. What does this method even do?

Reply:
Opps, forgot to actually remove these lines. I have no idea why the free_symbols method was written to return `{self}`. The PR that did that didn't have any useful comments about it. New commit removes the lines as intended.